recipe/meta.yaml

```diff
@@ -19,17 +19,12 @@ requirements:
     - pip
   run:
     - python >=3.7
+    - geopandas
```

Comment:
You could also use `geopandas-base` here if you further want to limit the default set of packages (compared to `geopandas`, that would avoid fiona/gdal, mapclassify (which requires scikit-learn, networkx) and folium)

Reply:
Nice, let me try that.